.scalafmt.conf

```diff
@@ -0,0 +1,130 @@
+# Test upgrades: $ scripts/scalafmt --test 2> diff.txt
+version = "3.10.3+10-2b4fdf2e-SNAPSHOT"
```

Comment:
this is the most recent commit. if adjustments are requested and implemented, this will change. when this PR is about to be accepted, I will release and update to a released version.